shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -205,8 +246,13 @@ private String getAppBundlePath() {
    * The reason that a {@code <meta-data>} preference is supported is because this {@code Activity}
    * might be the very first {@code Activity} launched, which means the developer won't have
    * control over the incoming {@code Intent}.
+   *
+   * Subclasses may override this method to directly control whether or not a splash screen is
+   * displayed.
+   *
+   * TODO(mattcarroll): move all splash behavior to FlutterView
```

Comment:
Move TODOs to a non-doc comment since protected member docs end up on public doc pages.